runtime/ast/parameter.go

```diff
@@ -25,25 +25,28 @@ import (
 )
 
 type Parameter struct {
-	TypeAnnotation *TypeAnnotation
-	Label          string
-	Identifier     Identifier
-	StartPos       Position `json:"-"`
+	TypeAnnotation  *TypeAnnotation
+	DefaultArgument Expression
```

Comment:
This also needs to get included in `ParameterList.Doc()`. Maybe while at it, move the loop body, the Doc construction for `Parameter` out of the loop, and into a separate function `Parameter.Doc()`